src/org/traccar/protocol/AppelloProtocolDecoder.java

```diff
@@ -36,8 +35,8 @@ public AppelloProtocolDecoder(AppelloProtocol protocol) {
             .text("FOLLOWIT,")                   // brand
             .number("(d+),")                     // imei
             .groupBegin()
-            .number("(dd)(dd)(dd)")              // date
-            .number("(dd)(dd)(dd).?d*,")         // time
+            .number("(dd)(dd)(dd)")              // date (yymmdd)
+            .number("(dd)(dd)(dd).?d*,")         // time (hhmmss.ms)
```

Comment:
I think using "ms" is not correct here. Just remove milliseconds from format comment, because we are not capturing it anyway. Time format is rarely useful because the order is always the same anyway. Same for other similar cases.